lib/cm3/itm.c

```diff
@@ -0,0 +1,156 @@
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2019
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+@section  itm_example Example ITM setup for async SWO
+
+Example: Async SWO output enabled, Manchester format, 8MHz:
+@code
+  // Enable trace subsystem
+  scs_set_trace_enabled(true);
+
+  // Use Manchester code for asynchronous transmission
+  tpiu_set_trace_output_mode(TPIU_SPPR_ASYNC_MANCHESTER);
```

Comment:
If you're writing examples, a) still use C style comments, and b) use an example that's useful to people, so that's not likely to be manchester.